emData/download.sh

```diff
@@ -3,7 +3,7 @@ set -e
 
 #### generated with commit 6e61462 ####
```

Comment:
@mlarson02 Could you please update this comment with the commit that was used to generate the test vectors? If different commits were used to generate different sets of test vectors, you should indicate that below, e.g.:
```bash
…
# Combined modules (generated with commit fffffff)
…
# Reduced Combined modules (generated with commit eeeeeee)
…
```

This will all wash out once we're able to do a full synchronization with the emulation, but in the meantime, we should keep track of how these files were generated.